onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -300,9 +300,10 @@ public void onInfoWindowClick(Marker marker) {
         ObaTripStatus status = mMarkerData.getStatusFromMarker(marker);
 
         if (mController != null && mController.getFocusedStopId() != null) {
-            TripDetailsActivity.start(mActivity, status.getActiveTripId(), mController.getFocusedStopId());
+            TripDetailsActivity.start(mActivity, status.getActiveTripId(),
+                    mController.getFocusedStopId(), "vehicle");
         } else {
-            TripDetailsActivity.start(mActivity, status.getActiveTripId());
+            TripDetailsActivity.start(mActivity, status.getActiveTripId(), null, "vehicle");
```

Comment:
I'd prefer creating a new overloaded `TripDetailsActivity.start()` method without stopId so we don't have to pass in `null` here.